android/app/src/main/java/org/fossasia/openevent/core/track/session/SessionDetailActivity.java

```diff
@@ -225,6 +225,7 @@ private void updateSession() {
         SessionType type = session.getSessionType();
 
         if (type != null) {
+            typeLabel.setVisibility(View.VISIBLE);
             sessionType.setText(type.getName());
             typelayout.setVisibility(View.VISIBLE);
```

Comment:
When type layout is shown and hidden, why is there a need to show and hide textview, please make it only one